awx/ui/src/screens/TopologyView/utils/helpers.js

```diff
@@ -128,7 +130,8 @@ export const generateRandomNodes = (n) => {
       'installed',
       'provision-fail',
       'deprovision-fail',
-    ][getRandomInt(0, 5)];
+      'unavailable',
+    ][getRandomInt(0, 6)];
```

Comment:
Just curious, what does the getRandomInt do here?

Reply:
It just returns a random number between 0-6. So the state array randomly returns a state by calling it like so: `['ready', 'installed', etc.][2]`